modules/gcp/vpc/variables.tf

```diff
@@ -0,0 +1,67 @@
+variable "project_id" {
```

Comment:
Always accept a set of tags/labels to all gcp modules. This is added all gcp resource/module block in the terraform